website/www/site/content/en/contribute/release-guide.md

```diff
@@ -471,6 +471,11 @@ For all other JIRA issues:
 
 If there is a bug found in the RC creation process/tools, those issues should be considered high priority and fixed in 7 days.
 
+### Process for cherry-picks
```

Comment:
Nit: I don't think we need a new section for this, we can just add the first bullet point to the existing "Review cherry-picks" section.

Reply:
Sounds Good!